src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -59,12 +59,12 @@ typedef enum {
  * Reference: https://www.w3.org/TR/webrtc-stats/#rtcstatsicecandidatepairstate-enum
  */
 typedef enum {
-    RTC_ICE_CANDIDATE_PAIR_STATE_FROZEN         = 0,
-    RTC_ICE_CANDIDATE_PAIR_STATE_WAITING        = 1,
-    RTC_ICE_CANDIDATE_PAIR_STATE_IN_PROGRESS    = 2,
-    RTC_ICE_CANDIDATE_PAIR_STATE_SUCCEEDED      = 3,
-    RTC_ICE_CANDIDATE_PAIR_STATE_FAILED         = 4,
-} RTC_ICE_CANDIDATE_PAIR_STATE;
+    ICE_CANDIDATE_PAIR_STATE_FROZEN      = 0,
```

Comment:
code refers to `ICE_CANDIDATE_PAIR_*` not `RTC_ICE_CANDIDATE_PAIR_*`

Reply:
The reason I added RTC_ is to avoid conflict with the internal state enum. I will make the relevant changes when I add ICE candidate related metrics.  Let us not mix this with the RTP stats. :)